app/Services/ReservationReassignmentService.php

```
@@ -503,13 +503,29 @@ private function notifyUserCopyUnavailable(int $prestitoId, string $reason): voi
+        // flushDeferredNotifications dopo il commit), quindi nessuna I/O in transazione.
+        try {
+            $this->notificationService->sendCopyUnavailableNotification($data['email'], [
+                'utente_nome' => $data['utente_nome'],
+                'libro_titolo' => $data['libro_titolo'],
+                'motivo' => $reasonText,
+            ]);
+        } catch (\Throwable $e) {
+            SecureLogger::warning(__('Email copia non disponibile fallita'), [
+                'prestito_id' => $prestitoId,
+                'error' => $e->getMessage()
+            ]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\SecureLogger' in method 'notifyUserCopyUnavailable'. (undefined)

(StaticAccess)